### ISSUE TITLE
Add option to set axis labels to vertical, horizontal and angle

### ISSUE THREAD

**Issue by lauramurgatroyd** [TomographicImaging/CILViewer#122]
We can have an option to set whether the dataset is acquisition data, in which case the labels on the orientation marker will be changed.

**Comment by paskino** [comment]
This should use `SetXAxisLabelText` from the `vtkAxesActor` that can be obtained as `prop` by 

```python
v = viewer2D() # or viewer3D
prop = v.GetOrientationMarker()
```

The `vtkAxerActor` is defined here in `viewer3D` https://github.com/vais-ral/CILViewer/blob/dcea26741eafc49b5799ac8b817bb7fab51c6800/Wrappers/Python/ccpi/viewer/CILViewer.py#L392-L399 or in `viewer2D` https://github.com/vais-ral/CILViewer/blob/dcea26741eafc49b5799ac8b817bb7fab51c6800/Wrappers/Python/ccpi/viewer/CILViewer2D.py#L1356-L1365

**Comment by lauramurgatroyd** [comment]
WIP on [this branch](https://github.com/vais-ral/CILViewer/tree/axis_labels), I'll come back to fixing the text size:

![image](https://user-images.githubusercontent.com/60604372/135460449-f84cba12-2816-4c1c-b43a-b690330f87eb.png)

**Comment by DanicaSTFC** [comment]
I will take over this issue and the branch

**Comment by DanicaSTFC** [comment]
This issue will be partially closed by #389 and will be finished in #398
